<!-- Output copied to clipboard! -->

<!-----

Yay, no errors, warnings, or alerts!

Conversion time: 3.064 seconds.


Using this Markdown file:

1. Paste this output into your source file.
2. See the notes and action items below regarding this conversion run.
3. Check the rendered output (headings, lists, code blocks, tables) for proper
   formatting and use a linkchecker before you publish this page.

Conversion notes:

* Docs to Markdown version 1.0Î²34
* Wed Jun 14 2023 10:50:11 GMT-0700 (PDT)
* Source doc: CV
* Tables are currently converted to HTML tables.
----->



# William S. Sanders


### Employment


<table>
  <tr>
   <td>Director, Research IT - 
<p>
Infrastructure & Data Services
   </td>
   <td>Information Technology
<p>
The Jackson Laboratory
<p>
Farmington, CT
   </td>
   <td><p style="text-align: right">
Nov-2021 to Present</p>

   </td>
  </tr>
  <tr>
   <td>Senior Manager - 
<p>
Research Cyberinfrastructure
   </td>
   <td>Information Technology 
<p>
The Jackson Laboratory
<p>
Farmington, CT
   </td>
   <td><p style="text-align: right">
Jul-2021 to Nov-2021</p>

   </td>
  </tr>
  <tr>
   <td>Manager - 
<p>
Research Cyberinfrastructure
   </td>
   <td>Information Technology
<p>
The Jackson Laboratory
<p>
Farmington, CT
   </td>
   <td><p style="text-align: right">
Jan-2018 to Jul-2021</p>

   </td>
  </tr>
  <tr>
   <td>Systems Analyst - 
<p>
High Performance Computing
   </td>
   <td>Information Technology 
<p>
The Jackson Laboratory
<p>
Farmington, CT
   </td>
   <td><p style="text-align: right">
Feb-2016 to Jan-2018</p>

   </td>
  </tr>
  <tr>
   <td>Section Editor (Bioinformatics)
   </td>
   <td>Genomics, Proteomics, and Bioinformatics
<p>
Beijing Institute of Genomics
<p>
Chinese Academy of Sciences
<p>
Beijing, China
   </td>
   <td><p style="text-align: right">
Apr-2014 to Jun-2015</p>

   </td>
  </tr>
  <tr>
   <td>Postdoctoral Associate - 
<p>
Peterson Laboratory
   </td>
   <td>Institute for Genomics, Biocomputing, and Biotechnology
<p>
Mississippi State University
<p>
Starkville, MS
   </td>
   <td><p style="text-align: right">
Oct-2011 to Feb-2016</p>

   </td>
  </tr>
  <tr>
   <td>Programmer / Analyst
   </td>
   <td>Institute for Genomics, Biocomputing, and Biotechnology
<p>
Mississippi State University
<p>
Starkville, MS
   </td>
   <td><p style="text-align: right">
May-2011 to Oct-2011</p>

   </td>
  </tr>
  <tr>
   <td>Graduate Research Assistant
   </td>
   <td>Department of Computer Science & Engineering
<p>
Mississippi State University
<p>
Starkville, MS
   </td>
   <td><p style="text-align: right">
Jan-2007 to May-2011</p>

   </td>
  </tr>
  <tr>
   <td>Graduate Research Assistant
   </td>
   <td>Department of Biochemistry & Molecular Biology
<p>
Mississippi State University
<p>
Starkville, MS
   </td>
   <td><p style="text-align: right">
Jan-2004 to Dec-2006</p>

   </td>
  </tr>
  <tr>
   <td>Teaching Assistant
   </td>
   <td>Department of Biochemistry & Molecular Biology
<p>
Mississippi State University
<p>
Starkville, MS
   </td>
   <td><p style="text-align: right">
Jun-2002 to May-2003</p>

   </td>
  </tr>
  <tr>
   <td>Computer & Network Technician
   </td>
   <td>Computers Plus
<p>
Oxford, MS
   </td>
   <td><p style="text-align: right">
Jul-1998 to Jun-1999</p>

   </td>
  </tr>
</table>



### Education


<table>
  <tr>
   <td>Ph.D.
   </td>
   <td>Molecular Biology
<p>
<em>Computational Biology & Machine Learning</em>
<p>
Advisor: Dr. Susan Bridges
   </td>
   <td>Mississippi State University
   </td>
   <td><p style="text-align: right">
2011</p>

   </td>
  </tr>
  <tr>
   <td>M.B.A.
   </td>
   <td>Business Administration
   </td>
   <td>University of Illinois at Urbana-Champaign
   </td>
   <td><p style="text-align: right">
2023</p>

   </td>
  </tr>
  <tr>
   <td>M.S.
   </td>
   <td>Computer Science
   </td>
   <td>Mississippi State University
   </td>
   <td><p style="text-align: right">
2020</p>

   </td>
  </tr>
  <tr>
   <td>B.S.
   </td>
   <td>Computer Science
   </td>
   <td>Mississippi State University
   </td>
   <td><p style="text-align: right">
2003</p>

   </td>
  </tr>
  <tr>
   <td>B.S.
   </td>
   <td>Biochemistry
   </td>
   <td>Mississippi State University
   </td>
   <td><p style="text-align: right">
2003</p>

   </td>
  </tr>
</table>



### Publications


<table>
  <tr>
   <td>NIH SenNet Consortium to map senescent cells throughout the human lifespan to understand physiological health
   </td>
   <td><strong>Nature Aging</strong>
   </td>
   <td><p style="text-align: right">
2022</p>

   </td>
  </tr>
  <tr>
   <td>Performance Modeling of Scalable Resource Allocations with the Imperial PEPA Compiler
   </td>
   <td><strong>IEEE 21st International Symposium on Parallel and Distributed Computing (ISPDC)</strong>
   </td>
   <td><p style="text-align: right">
2022</p>

   </td>
  </tr>
  <tr>
   <td>Addressing the Robustness of Resource Allocation in the Presence of Application and System Irregularities via PEPA Based Modeling
   </td>
   <td><strong>Lecture Notes in Computer Science (International Conference on Computational Science(ICCS))</strong>
   </td>
   <td><p style="text-align: right">
2020</p>

   </td>
  </tr>
  <tr>
   <td>Robustness Analysis of Scaled Resource Allocation Models Using the Imperial PEPA Compiler
   </td>
   <td><strong>IEEE 19th International Symposium on Parallel and Distributed Computing (ISPDC)</strong>
   </td>
   <td><p style="text-align: right">
2020</p>

   </td>
  </tr>
  <tr>
   <td>A Container-Based Framework to Facilitate Reproducibility in Employing Stochastic Process Algebra for Modeling Parallel Computing Systems
   </td>
   <td><strong>IEEE International Parallel and Distributed Processing Symposium Workshops</strong>
   </td>
   <td><p style="text-align: right">
2019</p>

   </td>
  </tr>
  <tr>
   <td>A genomic resource for the sedentary semi-endoparasitic reniform nematode, <em>Rotylenchulus reniformis</em> Linford & Oliveira
   </td>
   <td><strong> Journal of Nematology</strong>
   </td>
   <td><p style="text-align: right">
2019</p>

   </td>
  </tr>
  <tr>
   <td>Five life stage specific transcriptome assemblies for <em>Rotylenchulus reniformis</em>
   </td>
   <td><strong>Journal of Nematology</strong>
   </td>
   <td><p style="text-align: right">
2018</p>

   </td>
  </tr>
  <tr>
   <td>Insights into the evolution of the New World diploid cottons (<em>Gossypium</em>, subgenus <em>Houzingenia</em>) based on genome sequencing
   </td>
   <td><strong>Genome Biology and Evolution</strong>
   </td>
   <td><p style="text-align: right">
2018</p>

   </td>
  </tr>
  <tr>
   <td>Comparative genomics of an unusual biogeographic disjunction in the cotton tribe (Gossypieae) yields insights into genome downsizing
   </td>
   <td><strong>Genome Biology and Evolution</strong>
   </td>
   <td><p style="text-align: right">
2017</p>

   </td>
  </tr>
  <tr>
   <td>Independent Domestication of Two Old World Cotton Species
   </td>
   <td><strong>Genome Biology and Evolution</strong>
   </td>
   <td><p style="text-align: right">
2016</p>

   </td>
  </tr>
  <tr>
   <td>Comparative Analysis of Fresh and Cryopreserved Boar Spermatozoa Using RNA Sequencing
   </td>
   <td><strong>Reproduction, Fertility, and Development</strong>
   </td>
   <td><p style="text-align: right">
2015</p>

   </td>
  </tr>
  <tr>
   <td>Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibres
   </td>
   <td><strong>Nature</strong>
   </td>
   <td><p style="text-align: right">
2012</p>

   </td>
  </tr>
  <tr>
   <td>Prediction of Cell Penetrating Peptides by Support Vector Machines
   </td>
   <td><strong>PLoS Computational Biology</strong>
   </td>
   <td><p style="text-align: right">
2011</p>

   </td>
  </tr>
  <tr>
   <td>Machine Learning and Mapping Algorithms Applied to Proteomics Problems
   </td>
   <td><strong>ProQuest/UMI Dissertation Publishing</strong>
   </td>
   <td><p style="text-align: right">
2011</p>

   </td>
  </tr>
  <tr>
   <td>The Proteogenomic Mapping Tool
   </td>
   <td><strong>BMC Bioinformatics</strong>
   </td>
   <td><p style="text-align: right">
2011</p>

   </td>
  </tr>
  <tr>
   <td>Prediction of peptides observable by mass spectrometry applied at the experimental set level
   </td>
   <td><strong>BMC Bioinformatics</strong>
   </td>
   <td><p style="text-align: right">
2007</p>

   </td>
  </tr>
</table>



### Funding 


<table>
  <tr>
   <td><strong>Role</strong>
   </td>
   <td><strong>Title</strong>
   </td>
   <td><strong>Organization</strong>
   </td>
   <td><strong>Amount</strong>
   </td>
   <td><strong>Date</strong>
   </td>
   <td><strong>Status</strong>
   </td>
  </tr>
  <tr>
   <td>Key
   </td>
   <td>U24 - Cellular Senescence Network: Consortium Organization and Data Coordinating Center
   </td>
   <td>NIH
   </td>
   <td>-
   </td>
   <td>2021-2026
   </td>
   <td><strong>Funded</strong>
   </td>
  </tr>
  <tr>
   <td>Key
   </td>
   <td>U54 - The KAPP-Sen Tissue Mapping Center Collaborative
   </td>
   <td>NIH
   </td>
   <td>-
   </td>
   <td>2021-2026
   </td>
   <td><strong>Funded</strong>
   </td>
  </tr>
  <tr>
   <td>PI
   </td>
   <td>JAX Research Computing Community Assessment
   </td>
   <td>JAX
   </td>
   <td>$77,185.56
   </td>
   <td>2021
   </td>
   <td><strong>Funded</strong>
   </td>
  </tr>
  <tr>
   <td>PI
   </td>
   <td>Finishing the <em>Crotalus horridus</em> (Timber Rattlesnake) Draft Genome with the MidSouth Computational Biology and Bioinformatics Society
   </td>
   <td>MSU-IGBB, MSU-CSE, MSU-CVM
   </td>
   <td>$10,224.28
   </td>
   <td>2015
   </td>
   <td><strong>Funded</strong>
   </td>
  </tr>
  <tr>
   <td>PI
   </td>
   <td>Sequencing and Assembling a Draft Genome of <em>Kokia drynarioides</em>, a <em>Gossypium</em> Outgroup
   </td>
   <td>MSU-IGBB, USDA-FAS-Borlaug
   </td>
   <td>$12,962.00
   </td>
   <td>2015
   </td>
   <td><strong>Funded</strong>
   </td>
  </tr>
  <tr>
   <td>Co-PI
   </td>
   <td>Developing Resources and Tools to Facilitate Discovery of Disease-Associated microRNAs in Farm Animals
   </td>
   <td>USDA-NIFA
   </td>
   <td>$448,158.00
   </td>
   <td>2014
   </td>
   <td>Not Funded
   </td>
  </tr>
  <tr>
   <td>PostDoc
   </td>
   <td>Genomic Analysis of Cultivated Cotton and Related Wild Species
   </td>
   <td>USDA ARS
   </td>
   <td>-
   </td>
   <td>2011-2016
   </td>
   <td><strong>Funded</strong>
   </td>
  </tr>
  <tr>
   <td>PostDoc
   </td>
   <td>Increasing US Cotton Competitiveness Through Genomics
   </td>
   <td>USDA ARS
   </td>
   <td>-
   </td>
   <td>2007-2012
   </td>
   <td><strong>Funded</strong>
   </td>
  </tr>
</table>



### Service


<table>
  <tr>
   <td>Scientific Track Chair - Panels
   </td>
   <td><strong>Practice and Experience in Advanced Research Computing (PEARC) Conference - </strong>2023
<p>
Portland, OR
   </td>
   <td><p style="text-align: right">
2023</p>

   </td>
  </tr>
  <tr>
   <td>Campus Champions Leadership Team (Elected Member)
   </td>
   <td><strong>Campus Champions https://campuschampions.cyberinfrastructure.org/</strong>
   </td>
   <td><p style="text-align: right">
2021-2023</p>

   </td>
  </tr>
  <tr>
   <td>Scientific Track Chair -
<p>
Advanced Research Computing Software and Applications
   </td>
   <td><strong>Practice and Experience in Advanced Research Computing (PEARC) Conference - </strong>2019
<p>
Chicago, IL
   </td>
   <td><p style="text-align: right">
2019</p>

   </td>
  </tr>
  <tr>
   <td>Technical Committee Member
   </td>
   <td><strong>Practice and Experience in Advanced Research Computing (PEARC) Conference</strong> - 2018
<p>
Pittsburgh, PA
   </td>
   <td><p style="text-align: right">
2017-2019</p>

   </td>
  </tr>
  <tr>
   <td>Organizing Committee Member
   </td>
   <td><strong>MidSouth Computational Biology and Bioinformatics Society Annual Conference</strong> - 2018
<p>
Starkville, MS
   </td>
   <td><p style="text-align: right">
2017-2018</p>

   </td>
  </tr>
  <tr>
   <td>Campus Champion
<p>
The Jackson Laboratory
   </td>
   <td><strong>Campus Champions - Extreme Science and Engineering Discovery Environment (XSEDE)</strong>
<p>
National Science Foundation
   </td>
   <td><p style="text-align: right">
2016-Present</p>

   </td>
  </tr>
  <tr>
   <td>Workshop Organizer -  
<p>
Evolution of Genome Size
   </td>
   <td><strong>Plant and Animal Genome Conference</strong>
<p>
San Diego, CA
   </td>
   <td><p style="text-align: right">
2016-2020</p>

   </td>
  </tr>
  <tr>
   <td>Project Leader - <em>Crotalus horridus</em> (Timber Rattlesnake) Genome Project
   </td>
   <td><strong>MidSouth Computational Biology and Bioinformatics Society</strong>
   </td>
   <td><p style="text-align: right">
2012-2018</p>

   </td>
  </tr>
  <tr>
   <td>Session Chair - Proteomics Session
   </td>
   <td><strong>MidSouth Computational Biology and Bioinformatics Society Annual Conference</strong>
<p>
Oxford, MS
   </td>
   <td><p style="text-align: right">
2012</p>

   </td>
  </tr>
</table>



### Certifications, Trainings, & Licensures


<table>
  <tr>
   <td>Gies College of Business
<p>
University of Illinois at Urbana-Champaign
   </td>
   <td>Entrepreneurship and Strategic Innovation
   </td>
   <td><p style="text-align: right">
2023</p>

   </td>
  </tr>
  <tr>
   <td>ROI Training
   </td>
   <td>Google Cloud Platform - Professional Cloud Architect Training
   </td>
   <td><p style="text-align: right">
2022</p>

   </td>
  </tr>
  <tr>
   <td>SeaChange Resources
   </td>
   <td>MaritimeQuest / Executive Leadership Program
   </td>
   <td><p style="text-align: right">
2022</p>

   </td>
  </tr>
  <tr>
   <td>Gies College of Business
<p>
University of Illinois at Urbana-Champaign
   </td>
   <td>Financial Management
   </td>
   <td><p style="text-align: right">
2022</p>

   </td>
  </tr>
  <tr>
   <td>Gies College of Business
<p>
University of Illinois at Urbana-Champaign
   </td>
   <td>Strategic Leadership & Management
   </td>
   <td><p style="text-align: right">
2022</p>

   </td>
  </tr>
  <tr>
   <td>Dr. Daylian M. Cain
<p>
Negotiation Mind Games
   </td>
   <td>Negotiating in Difficult Times
   </td>
   <td><p style="text-align: right">
2021</p>

   </td>
  </tr>
  <tr>
   <td>KU Leuven / EdX
   </td>
   <td>Beer: The Science of Brewing
   </td>
   <td><p style="text-align: right">
2021</p>

   </td>
  </tr>
  <tr>
   <td>SeaChange Resources
   </td>
   <td>Leadership@theHelm / Crisis Management
   </td>
   <td><p style="text-align: right">
2021</p>

   </td>
  </tr>
  <tr>
   <td>The Jackson Laboratory & Johns Hopkins University
   </td>
   <td>Human and Mammalian Genetics and Genomics: The 61st McKusick Short Course 
   </td>
   <td><p style="text-align: right">
2020</p>

   </td>
  </tr>
  <tr>
   <td>Scaled Agile
   </td>
   <td>Certified SAFe 4 DevOps Practitioner 
   </td>
   <td><p style="text-align: right">
2018</p>

   </td>
  </tr>
  <tr>
   <td>Scaled Agile
   </td>
   <td>Leading SAFe (Certified SAFe 4 Agilist)
   </td>
   <td><p style="text-align: right">
2018</p>

   </td>
  </tr>
  <tr>
   <td>The Jackson Laboratory
   </td>
   <td>Frontline Leadership Training
   </td>
   <td><p style="text-align: right">
2018</p>

   </td>
  </tr>
  <tr>
   <td>Karrass
   </td>
   <td>Effective Negotiating
   </td>
   <td><p style="text-align: right">
2017</p>

   </td>
  </tr>
  <tr>
   <td>Karrass
   </td>
   <td>Negotiating Effectively Within Your Own Organization
   </td>
   <td><p style="text-align: right">
2017</p>

   </td>
  </tr>
  <tr>
   <td>American Red Cross
   </td>
   <td>Adult CPR/AED, Infant CPR, and First Aid
<p>
<em>(GVCZ9G)</em>
   </td>
   <td><p style="text-align: right">
2017-2019</p>

   </td>
  </tr>
  <tr>
   <td>Software Carpentry Foundation
   </td>
   <td>Software Carpentry Certified Instructor
   </td>
   <td><p style="text-align: right">
2016</p>

   </td>
  </tr>
  <tr>
   <td>ROI Training
   </td>
   <td>Google Cloud Platform Fundamentals Certificate (<em>CP100A</em>)
   </td>
   <td><p style="text-align: right">
2016</p>

   </td>
  </tr>
  <tr>
   <td>Bagley College of Engineering
<p>
Mississippi State University
   </td>
   <td>Information Assurance Certificate
<p>
<em>(NSTISSI #4011 & NSTISSI #4014 Certified Program)</em>
   </td>
   <td><p style="text-align: right">
2010</p>

   </td>
  </tr>
  <tr>
   <td>Federal Communications Commission
<p>
Amateur Radio Service
   </td>
   <td>Amateur Radio Technician License
<p>
<em>(Callsign: KF5CZK)</em>
   </td>
   <td><p style="text-align: right">
2009</p>

   </td>
  </tr>
  <tr>
   <td>Bruker Daltonics
   </td>
   <td>FLEX Series MALDI-TOF MS Operator Course
   </td>
   <td><p style="text-align: right">
2006</p>

   </td>
  </tr>
  <tr>
   <td>Bagley College of Engineering
<p>
Mississippi State University
   </td>
   <td>Computational Biology Certificate
   </td>
   <td><p style="text-align: right">
2007</p>

   </td>
  </tr>
  <tr>
   <td>Microsoft
   </td>
   <td>Microsoft Certified Professional
<p>
<em>Windows NT 4.0, Networking Essentials</em>
   </td>
   <td><p style="text-align: right">
1998-2001</p>

   </td>
  </tr>
  <tr>
   <td>CompTIA
   </td>
   <td>CompTIA A+ Certified Professional <em>(COMP10072148)</em>
   </td>
   <td><p style="text-align: right">
1998</p>

   </td>
  </tr>
</table>



### Awards & Honors


<table>
  <tr>
   <td>JAX Champion - Excellence in Quality and Innovation Award
   </td>
   <td>The Jackson Laboratory
<p>
Bar Harbor, ME | Farmington, CT | Sacramento, CA
   </td>
   <td><p style="text-align: right">
2018</p>

   </td>
  </tr>
  <tr>
   <td>1<sup>st</sup> Place Postdoctoral Oral Presentation
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Little Rock, AR
   </td>
   <td><p style="text-align: right">
Mar 13-14 2015</p>

   </td>
  </tr>
  <tr>
   <td>3<sup>rd</sup> Place Student Oral Presentation
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Oklahoma City, OK
   </td>
   <td><p style="text-align: right">
Feb 23-24 2008</p>

   </td>
  </tr>
  <tr>
   <td>1<sup>st</sup> Place Student Poster Presentation
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
New Orleans, LA
   </td>
   <td><p style="text-align: right">
Feb 1-3 2007</p>

   </td>
  </tr>
  <tr>
   <td>University Honors Program - Phase I Scholar
   </td>
   <td>Mississippi State University
<p>
Starkville, MS
   </td>
   <td><p style="text-align: right">
May 2000</p>

   </td>
  </tr>
</table>



### Teaching Experience


<table>
  <tr>
   <td>Instructor
   </td>
   <td>Introduction to Application Containerization Using Singularity
<p>
12th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics
<p>
Chicago, IL
   </td>
   <td><p style="text-align: right">
2021</p>

   </td>
  </tr>
  <tr>
   <td>Instructor
   </td>
   <td>Python and R for Scientific Computing
<p>
Software Carpentry Workshop
<p>
The Jackson Laboratory
   </td>
   <td><p style="text-align: right">
2019</p>

   </td>
  </tr>
  <tr>
   <td>Instructor
   </td>
   <td>Introduction to High Performance Computing Workshop
<p>
The Jackson Laboratory
   </td>
   <td><p style="text-align: right">
2016-2019</p>

   </td>
  </tr>
  <tr>
   <td>Instructor
   </td>
   <td>Excel, OpenRefine, Python
<p>
Data Carpentry Workshop
<p>
The Jackson Laboratory
   </td>
   <td><p style="text-align: right">
2017</p>

   </td>
  </tr>
  <tr>
   <td>Instructor
   </td>
   <td>Cloud Computing for Genomics
<p>
Data Carpentry Workshop
<p>
The Jackson Laboratory
   </td>
   <td><p style="text-align: right">
2017</p>

   </td>
  </tr>
  <tr>
   <td>Instructor
   </td>
   <td>R for Reproducible Scientific Analysis
<p>
Software Carpentry Workshop
<p>
The Jackson Laboratory
   </td>
   <td><p style="text-align: right">
2016</p>

   </td>
  </tr>
  <tr>
   <td>Mentor
   </td>
   <td>Day One Leadership Community
<p>
Mississippi State University
   </td>
   <td><p style="text-align: right">
2013-2015</p>

   </td>
  </tr>
  <tr>
   <td>Guest Lecturer
   </td>
   <td>High Throughput Sequence Analysis (<em>CSE 6990</em>)
<p>
Department of Computer Science & Engineering
<p>
Mississippi State University
   </td>
   <td><p style="text-align: right">
2015</p>

   </td>
  </tr>
  <tr>
   <td>Mentor
   </td>
   <td>Norman E. Borlaug International Agricultural Science and Technology Fellowship Program
<p>
Foreign Agricultural Service
<p>
United States Department of Agriculture
   </td>
   <td><p style="text-align: right">
2014-2015</p>

   </td>
  </tr>
  <tr>
   <td>Mentor
   </td>
   <td>Research Experience for Undergraduates
<p>
Undergraduate Research in Computational Biology
<p>
Mississippi State University
   </td>
   <td><p style="text-align: right">
2007-2015</p>

   </td>
  </tr>
 <tr>
   <td>Lecturer
   </td>
   <td>Biochemistry Boot Camp
<p>
Department of Chemistry
<p>
Mississippi State University
   </td>
   <td><p style="text-align: right">
2014-2015</p>

   </td>
  </tr>
  <tr>
   <td>Guest Lecturer
   </td>
   <td>Genomes & Genomics (<em>BCH 8653</em>)
<p>
Department of Biochemistry & Molecular Biology
<p>
Mississippi State University
   </td>
   <td><p style="text-align: right">
2012</p>

   </td>
  </tr>
  <tr>
   <td>Student
   </td>
   <td>Tools for Teaching Introductory Computer Science (<em>CSE 8990</em>)
<p>
Department of Computer Science & Engineering
<p>
Mississippi State University
   </td>
   <td><p style="text-align: right">
2010</p>

   </td>
  </tr>
  <tr>
   <td>Guest Lecturer
   </td>
   <td>Essentials of Molecular Genetics (<em>BCH 4113</em>)
<p>
Department of Biochemistry & Molecular Biology
<p>
Mississippi State University
   </td>
   <td><p style="text-align: right">
2009</p>

   </td>
  </tr>
  <tr>
   <td>Teaching Assistant
   </td>
   <td>Protein Methods (<em>BCH 4414/6414</em>)
<p>
Department of Biochemistry & Molecular Biology
<p>
Mississippi State University
   </td>
   <td><p style="text-align: right">
2004</p>

   </td>
  </tr>
  <tr>
   <td>Teaching Assistant
   </td>
   <td>Biochemical Methods (<em>BCH 4805/6805</em>)
<p>
Department of Biochemistry & Molecular Biology
<p>
Mississippi State University
   </td>
   <td><p style="text-align: right">
2002</p>

   </td>
  </tr>
</table>



### Reviewing Activities


<table>
  <tr>
   <td>Gene
   </td>
   <td>Nucleic Acids Research
   </td>
   <td>Bioinformatics
   </td>
  </tr>
  <tr>
   <td>Interdisciplinary Sciences: Computational Life Sciences
   </td>
   <td>Genomics, Proteomics, and Bioinformatics
   </td>
   <td>BMC Genomics
   </td>
  </tr>
  <tr>
   <td>PLoS One
   </td>
   <td>BMC Bioinformatics
   </td>
   <td>Journal of Translational Medicine
   </td>
  </tr>
  <tr>
   <td>Journal of Biocomputing
   </td>
   <td>BioSystems
   </td>
   <td>
   </td>
  </tr>
</table>



### Software Tools


<table>
  <tr>
   <td><strong>The Proteogenomic Mapping Tool.</strong>
<p>
https://agbase.arizona.edu/cgi-bin/tools/
   </td>
   <td>A Java-based implementation of the Aho-Corasick string searching algorithm which takes peptides observed by mass spectrometry and maps them onto a source genome.
   </td>
  </tr>
  <tr>
   <td><strong>PepFly </strong>â Peptide Flyability Prediction.
<p>
https://agbase.arizona.edu/cgi-bin/tools/
   </td>
   <td>Constructs a dataset for easy importation into the Weka machine learning toolkit from MS datasets to predict the flyability of peptides.
   </td>
  </tr>
</table>



### Skills


<table>
  <tr>
   <td><strong>Software Skills</strong>
   </td>
   <td>Programming Languages: Python, C/C++, Perl, Java, MPI
<p>
Statistical Analysis: R, SAS
<p>
Other: MySQL, WEKA Machine Learning Toolkit, Apache, BioPerl, BioPython, ABySS Sequence Assembler, Velvet Sequence Assembler, Roche 454 GS De Novo Assembler, CLC Genomics Workbench
   </td>
  </tr>
  <tr>
   <td><strong>Research Skills</strong>
   </td>
   <td>Genomics, Genome Sequence Assembly & Analysis, Proteomics, Mass Spectrometry, Artificial Intelligence, Machine Learning, Data mining, Algorithm Analysis & Design, Eukaryotic Tissue Culture, High Performance Computing, High Performance Liquid Chromatography, Gas Chromatography, Gel Electrophoresis
   </td>
  </tr>
  <tr>
   <td><strong>Miscellaneous</strong>
   </td>
   <td>German (minimal professional proficiency), Grant Writing
   </td>
  </tr>
</table>



### Presentations


<table>
  <tr>
   <td>Campus Champions: Connecting Community and Resources
   </td>
   <td>Practice and Experience in Advanced Research Computing (PEARC) 2022
<p>
Boston, MA
   </td>
   <td><p style="text-align: right">
Jul 11 2022</p>

   </td>
  </tr>
  <tr>
   <td>Words Matter! Promoting Inclusion through Language in Advanced Research Computing [Panelist]
   </td>
   <td>Supercomputing 2021
<p>
St. Louis, MO
   </td>
   <td><p style="text-align: right">
Nov 18 2021</p>

   </td>
  </tr>
  <tr>
   <td>Genomic Data Lakes: Research Data Storage at The Jackson Laboratory
   </td>
   <td>2021 Bio-IT World Conference and Expo
<p>
Boston, MA
   </td>
   <td><p style="text-align: right">
Sep 9 2021</p>

   </td>
  </tr>
  <tr>
   <td>Bursts, Forecasting, and Storm Avoidance: Utilization of Cloud Platforms
   </td>
   <td>Partly Cloudy 2019
<p>
Fred Hutchinson Cancer Research Center
<p>
Seattle, WA
   </td>
   <td><p style="text-align: right">
Oct 18 2019</p>

   </td>
  </tr>
  <tr>
   <td>Making It Work: Facilitating Collaboration Between IT and Researchers for Computation-Intensive Programs
   </td>
   <td>University of Mississippi Medical Center (UMMC) Bioinformatics Collaborative
<p>
Jackson, MS
   </td>
   <td><p style="text-align: right">
Mar 28 2018</p>

   </td>
  </tr>
  <tr>
   <td>A Mitochondrial Genome of the Timber Rattlesnake (<em>Crotalus horridus</em>) and a Reptilian Mitochondrial Phylogeny
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Memphis, TN
   </td>
   <td><p style="text-align: right">
Mar 3-5 2016</p>

   </td>
  </tr>
  <tr>
   <td>Genome size evolution among closely related species in <em>Gossypium</em>
   </td>
   <td>Plant and Animal Genome Conference XXIV
<p>
San Diego, CA
   </td>
   <td><p style="text-align: right">
Jan 9-13 2016</p>

   </td>
  </tr>
  <tr>
   <td>Genome Assembly & Analysis â From âGold-Standardâ Assemblies to âBase Metalsâ
<p>
<strong>Kriton-Hatzio Symposium Keynote Address</strong>
   </td>
   <td>Southern Section of the American Society of Plant Biologists (SS-ASPB)
<p>
Dauphin Island, AL
   </td>
   <td><p style="text-align: right">
Mar 28-29 2015</p>

   </td>
  </tr>
  <tr>
   <td>Assembly and molecular characterization of a draft genome sequence and annotation of the plant-parasitic nematode <em>Rotylenchulus reniformis</em>
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Little Rock, AR
   </td>
   <td><p style="text-align: right">
Mar 13-14 2015</p>

   </td>
  </tr>
  <tr>
   <td>Update on the Timber Rattlesnake Genome Collaborative Project
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Little Rock, AR
   </td>
   <td><p style="text-align: right">
Mar 13-14 2015</p>

   </td>
  </tr>
  <tr>
   <td>Draft Genome Assembly and Comparative Genomic Analysis of the Plant-Parasitic Nematode <em>Rotylenchulus reniformis</em>
   </td>
   <td>Plant and Animal Genome Conference XXIII
<p>
San Diego, CA
   </td>
   <td><p style="text-align: right">
Jan 10-14 2015</p>

   </td>
  </tr>
  <tr>
   <td>Molecular Insights into the <em>Rotylenchulus reniformis</em> transcriptome
   </td>
   <td>Plant and Animal Genome Conference XXIII
<p>
San Diego, CA
   </td>
   <td><p style="text-align: right">
Jan 10-14 2015</p>

   </td>
  </tr>
  <tr>
   <td>MCBIOS Timber Rattlesnake Genome Project Collaboration
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Stillwater, OK
   </td>
   <td><p style="text-align: right">
Mar 6-8 2014</p>

   </td>
  </tr>
  <tr>
   <td>Sequencing, Assembly, and Characterization of the Genome of <em>Rotylenchulus reniformis</em>
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Stillwater, OK
   </td>
   <td><p style="text-align: right">
Mar 6-8 2014</p>

   </td>
  </tr>
  <tr>
   <td>SABRe â Sequence Assembly By Reference
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Stillwater, OK
   </td>
   <td><p style="text-align: right">
Mar 6-8 2014</p>

   </td>
  </tr>
  <tr>
   <td>Assembly, Annotation and Proteogenomic Mapping of <em>Rotylenchulus reniformis</em>
   </td>
   <td>Plant and Animal Genome Conference XXII
<p>
San Diego, CA
   </td>
   <td><p style="text-align: right">
Jan 11-15 2014</p>

   </td>
  </tr>
  <tr>
   <td>Unveiling the Molecular Arsenal within <em>Rotylenchulus reniformis</em> via <em>de novo</em> Transcriptome Assembly.
   </td>
   <td>Annual Meeting of the Society of Nematologists
<p>
Knoxville, TN
   </td>
   <td><p style="text-align: right">
Jul 14-17 2013</p>

   </td>
  </tr>
  <tr>
   <td>Identification of Protein Coding Genes in the Plant-parasitic Nematode <em>Rotylenchulus reniformis</em> through Comparative Proteogenomic Mapping with <em>Caenorhabditis elegans</em>
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Columbia, MO
   </td>
   <td><p style="text-align: right">
Apr 5-6 2013</p>

   </td>
  </tr>
  <tr>
   <td>The Genome of Reniform Nematode, <em>Rotylenchulus reniformis</em>
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Columbia, MO
   </td>
   <td><p style="text-align: right">
Apr 5-6 2013</p>

   </td>
  </tr>
  <tr>
   <td><em>De Novo</em> Transcriptome Assembly of the Plant-Parasitic Nematode <em>Rotylenchulus reniformis</em>
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Columbia, MO
   </td>
   <td><p style="text-align: right">
Apr 5-6 2013</p>

   </td>
  </tr>
  <tr>
   <td>Genome Size Evolution In <em>Gossypium</em> (Cotton): Progress and Prospects
   </td>
   <td>Plant and Animal Genome Conference XXI
<p>
San Diego, CA
   </td>
   <td><p style="text-align: right">
Jan 12-16 2013</p>

   </td>
  </tr>
  <tr>
   <td>Assembling and Annotating the Genome of a Plant Parasitic Nematode, <em>Rotylenchulus reniformis</em>, with Sequences Obtained Using 2<sup>nd</sup> Generation Sequencing Technologies
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Oxford, MS
   </td>
   <td><p style="text-align: right">
Feb 17-18 2012</p>

   </td>
  </tr>
  <tr>
   <td>The <em>Rotylenchulus reniformis</em> Genome: Sequencing, Assembly, and Annotation
   </td>
   <td>Plant and Animal Genome Conference XX
<p>
San Diego, CA
   </td>
   <td><p style="text-align: right">
Jan 14-18 2012</p>

   </td>
  </tr>
  <tr>
   <td>Sequencing and Analysis of the <em>Rotylenchulus reniformis</em> Transcriptome
   </td>
   <td>Plant and Animal Genome Conference XX
<p>
San Diego, CA
   </td>
   <td><p style="text-align: right">
Jan 14-18 2012</p>

   </td>
  </tr>
  <tr>
   <td>Proteogenomic Mapping of <em>Gallus gallus</em> Serum
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
College Station, TX
   </td>
   <td><p style="text-align: right">
Apr 1-2 2011</p>

   </td>
  </tr>
  <tr>
   <td>Experimental genome annotation by proteogenomic mapping: experimental evidence for predicted proteins, novel proteins and protein structure
   </td>
   <td>Workshop on Nanotechnology, Proteogenomics and Visual Analytic Applications for Schistosomiasis Control
<p>
Abuja, Nigeria
   </td>
   <td><p style="text-align: right">
Dec 14-18 2010</p>

   </td>
  </tr>
  <tr>
   <td>Metrics for Mass Spectrometry Dataset Quality Validation
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
Jonesboro, AR
   </td>
   <td><p style="text-align: right">
Feb 19-20 2010</p>

   </td>
  </tr>
  <tr>
   <td>A Staged Search Strategy for Proteomics Based Structural Annotation in Chicken
   </td>
   <td>Plant and Animal Genome Conference XVIII
<p>
San Diego, CA
   </td>
   <td><p style="text-align: right">
Jan 9-13 2010</p>

   </td>
  </tr>
  <tr>
   <td>Proteomics Based Genome Annotation in <em>Gallus gallus</em>
   </td>
   <td>Mississippi State University Digital Biology Learning Community
<p>
Starkville, MS
   </td>
   <td><p style="text-align: right">
Oct 21 2009</p>

   </td>
  </tr>
  <tr>
   <td>Features Predictive of Cell Penetrating Peptides Validated by Machine Learning Classifiers
   </td>
   <td>Research in Computational Molecular Biology (RECOMB) 2009
<p>
Tucson, AZ
   </td>
   <td><p style="text-align: right">
May 18-21 2009</p>

   </td>
  </tr>
  <tr>
   <td>Prediction of Cell Penetrating Peptides by Machine Learning Classifiers
   </td>
   <td>Intelligent Systems for Molecular Biology (ISMB) 2008
<p>
Toronto, ON
   </td>
   <td><p style="text-align: right">
Jul 19-23 2008</p>

   </td>
  </tr>
  <tr>
   <td>Proteomics Based Structural Genome Annotation in Chicken
   </td>
   <td>Avian Genomics and Gene Ontology Annotation Workshop
<p>
Starkville, MS
   </td>
   <td><p style="text-align: right">
May 19-20 2008</p>

   </td>
  </tr>
  <tr>
   <td>Prospecting for new genes in the chicken genome using proteomics
   </td>
   <td>Cold Spring Harbor Laboratory - Genome Informatics Meeting
<p>
Cold Spring Harbor, NY
   </td>
   <td><p style="text-align: right">
Nov 1-5 2007</p>

   </td>
  </tr>
  <tr>
   <td>Pepfly: A Tool for the Prediction of Peptides Observed by Mass Spectra
   </td>
   <td>MidSouth Computational Biology and Bioinformatics Society Annual Conference
<p>
New Orleans, LA
   </td>
   <td><p style="text-align: right">
Feb 1-3 2007</p>

   </td>
  </tr>
</table>



### Miscellaneous



* OrcID: [https://orcid.org/my-orcid?orcid=0000-0001-9368-3004](https://orcid.org/my-orcid?orcid=0000-0001-9368-3004)
* Google Scholar: [https://scholar.google.com/citations?user=WIEDiZYAAAAJ&hl=en](https://scholar.google.com/citations?user=WIEDiZYAAAAJ&hl=en)
* GitHub: [https://github.com/williamssanders/public](https://github.com/williamssanders/public)
* LinkedIn: [https://www.linkedin.com/in/williamssanders/](https://www.linkedin.com/in/williamssanders/)
* JAX: [https://www.jax.org/people/shane-sanders](https://www.jax.org/people/shane-sanders)
